CMakeLists.txt

```diff
@@ -25,11 +25,6 @@ endif()
 
 include(${PROJECT_SOURCE_DIR}/toolchains/utils.cmake)
 
-# cmake output configuration
-set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
```

Comment:
It's unclear to me why we would want to use these custom output locations. I'd prefer to keep this change but you are correct that I need to update the README as well.